calculator/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Calculator</title>
+  <link rel="stylesheet" href="./style.css">
+  <script type="text/javascript" src="./math.js"></script>
+  <script type="text/javascript" src="./script.js"></script>
+ </head>
+<body>
+  <div class="calculator">
+    <div class="input-wrapper">
+      <input type="text" class="input-string" readonly>
+    </div>
+    <div class="buttons">
+      <div class="button-wrapper">
+        <button type="submit" class="calc-button btn-c">C</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-ce">CE</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-percent" value="%">%</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-div" value="/">/</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-7" value="7">7</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-8" value="8">8</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-9" value="9">9</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-mul" value="*">*</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-4" value="4">4</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-5" value="5">5</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-6" value="6">6</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-minus" value="-">-</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-1" value="1">1</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-2" value="2">2</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-3" value="3">3</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-plus" value="+">+</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-0" value="0">0</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-open-br" value="(">(</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-close-br" value=")">)</button>
+      </div>
+      <div class="button-wrapper">
+        <button class="calc-button btn-result">=</button>
+      </div>
```

Comment:
This is too much duplications, please try to add elements dynamically.
All you need 'C CE % / 7 8 9 * 4 5 6 - 1 2 3 + 0 ( ) ='.split(' ') and element for each element of the array